### ISSUE TITLE
Request for Guidance on Converting NOTALs to Presence in Syri Outputs

### ISSUE THREAD

**Issue by Lancer-sudo-png** [schneebergerlab/syri#246]
Dear @mnshgl0110,
I hope this message finds you well. I am reaching out to you because I am currently utilizing Syri, which I find to be an exceptionally useful tool for calling structural variants (SVs). However, I have encountered some challenges with presence/absence variations (PAVs) that I hope you can help clarify.

 After reviewing the discussions in the Issues section  [#107 ](https://github.com/schneebergerlab/syri/issues/107), I concur with the perspective that NOTALs could be considered as PAVs. My specific question concerns the process of converting a NOTAL, found in the query sequence, into a "Presence" status. In the Syri output file (syri.out), the locations are noted only for the query genome, with no corresponding descriptions for the reference genome,Like this:

![NOTAL-query pic](https://github.com/schneebergerlab/syri/assets/143490134/91e7aa47-14ec-46b9-961f-a6fc991a7fee)

Could you please advise on how to appropriately define a NOTAL-query as "Presence" in a VCF file format? Any guidance or suggestions you could provide would be greatly appreciated.

Thank you for your time and assistance.

Best regards,
Baoyue

**Comment by Lancer-sudo-png** [comment]
Thank you for your response! @mnshgl0110 

Indeed, I have identified a pattern based on adjacent annotation blocks: 

subtracting 1 from the left breakpoint of a NOTAL-query yields the right breakpoint of the adjacent block on the left. Using the right breakpoint of the adjacent block, I can locate its corresponding segment on the reference genome. Similarly, adding 1 to the right breakpoint of a NOTAL-query yields the left breakpoint of the adjacent block on the right, and I can locate its corresponding segment on the reference genome using this left breakpoint. 

However, I found that the segments on the reference genome identified using the adjacent block breakpoints may be located on different chromosomes. In such cases, how should I describe this NOTAL-query (i.e., what we consider as Presence) using the coordinates of the reference genome? Here is my variant information and a schematic diagram:
<img width="1086" alt="Pasted Graphic" src="https://github.com/schneebergerlab/syri/assets/143490134/807c00d1-5275-4a61-88f0-0942aa7c1812">
`NOTAL-query: -       -       -       -       -       Chr1    36665   37065   NOTAL27497      -       NOTAL   -`
`The left block of the NOTAL-query: Chr1 20046 29279   -  -  Chr1    27494 36664   SYN3 -  SYN -`
`The right block of the NOTAL-query: Chr3  4532378 4532617 - -  Chr1 37066 37306 INVDP47167  -  INVDP  copygain`

**Comment by mnshgl0110** [comment]
Indeed, that is the complication in describing Notals using reference genomes as sometimes there is no obvious answer. I guess, you can pick one convention for your analysis and then use that. Unfortunately, I do not have a more helpful answer.
